### ISSUE TITLE
Accuracy reaching 100%

### ISSUE THREAD

**Issue by taureanamir** [davidsandberg/facenet#956]
@davidsandberg 
I am training on my own dataset, 92 classes approx. 15 images per class, using softmax loss but the training accuracy is reaching 100%. this is very unusual. There must be something very fishy going on. Can you point me to the right direction?

**Comment by luameows** [comment]
I got the same problem as u did. While i trained webface dataset, the training acc still became 1. At first, i thought it was my dataset not balanced. Then, i did random flip for training set, and added dropout layer at the last fully connected layer, the situation didn't change.

**Comment by taureanamir** [comment]
@luameows 
Do you know if anyone else faced the similar issue??

**Comment by luameows** [comment]
@taureanamir 
As far as i know, ur dataset is too small for this deep network. Although softmax is not very strict for classification or verification, only 92*15 images can still cause overfit. Try to change a light CNN network or get more images for ur training may solver this situation.

**Comment by taureanamir** [comment]
@luameows 
Thank you very much for your suggestion.
